packages/manager/apps/telecom/src/app/telecom/pack/migration/buildings/pack-migration-buildings.html

```diff
@@ -0,0 +1,23 @@
+<div class="telecom-pack-migration-shipping">
+    <h3 data-translate="telecom_pack_migration_buildings_title"></h3>
+    <div data-translate="telecom_pack_migration_buildings_available"></div>
+    <oui-datagrid rows="$ctrl.buildingsList">
+        <oui-datagrid-column data-title="'telecom_pack_migration_buildings_name' | translate" data-property="name"
+            type="string">
+        </oui-datagrid-column>
+        <oui-datagrid-column data-title="'telecom_pack_migration_buildings_reference' | translate"
+            data-property="reference" type="string">
```

Comment:
```suggestion
            data-property="reference" data-type="string">
```
keep all `data-` or remove all `data-` to be consistent